jl4-cli/examples/not-ok/tests/parse-error1.ep.golden

```diff
@@ -0,0 +1,13 @@
+Info | updateFileDiagnostics published different from new diagnostics - file diagnostics:
+File:<<file>>
+  Hidden:   no
+  Range:    12:1-12:1
+  Source:   parser
+  Severity: DiagnosticSeverity_Error
+  Code:     <none>
+  Message: 
+    |
+    12 | GIVEN n IS A NUMBER
+    | ^^^^^
```

Comment:
Looks like something about the spacing / alignment of the errors is being broken. Not sure how easy this would be to fix.